conf/log4j2.xml.template

```diff
@@ -55,7 +55,7 @@
         <Root level="INFO">
             <AppenderRef ref="stdout"/>
         </Root>
-        <Logger name="org.apache.kyuubi.ctl.ServiceControlCli" level="error" additivity="false">
```

Comment:
This class has been removed

Reply:
since you created a separate `log4j2-repl.xml`, do we need such conf in `log4j2.xml` then?

Reply:
> since you created a separate `log4j2-repl.xml`, do we need such conf in `log4j2.xml` then?

I think it can be kept because user may not have created log4j2-repl.xml, in which case log4j2.xml can also be used.

Reply:
SGTM if it works, please manually check it

Reply:
> SGTM if it works, please manually check it

I have tested manually and it works fine.